Developer-Guide.md

```diff
@@ -301,6 +310,173 @@ $ sudo systemctl restart docker
+choose `runc`, but we also specify another container runtime, Kata Containers
+runtime in this case, as the runtime to run _untrusted_ workloads. In other
+words, this defines an alternative runtime that will be picked when the
+workload cannot be trusted and a higher level of security is required.
+An additional flag can be used to let CRI-O know if a workload should be
+considered _trusted_ or _untrusted_ by default.
+
+Additionally, we need the network namespace management to be performed by
+CRI-O, otherwise when the VM is started, the network has not been set and
+it will never get connected later.
+
+Here is an example of how you can modify `/etc/crio/crio.conf` file in order
+to apply explanations from above, and therefore get Kata Containers runtime
+being invoked by CRI-O.
+
+```toml
+# The "crio.runtime" table contains settings pertaining to the OCI
+# runtime used and options for how to set up and manage the OCI runtime.
+[crio.runtime]
+manage_network_ns_lifecycle = true
+
+# runtime is the OCI compatible runtime used for trusted container workloads.
+# This is a mandatory setting as this runtime will be the default one
+# and will also be used for untrusted container workloads if
+# runtime_untrusted_workload is not set.
+runtime = "/usr/bin/runc"
+
+# runtime_untrusted_workload is the OCI compatible runtime used for untrusted
+# container workloads. This is an optional setting, except if
+# default_container_trust is set to "untrusted".
+runtime_untrusted_workload = "/usr/local/bin/kata-runtime"
+
+# default_workload_trust is the default level of trust crio puts in container
+# workloads. It can either be "trusted" or "untrusted", and the default
+# is "trusted".
+# Containers can be run through different container runtimes, depending on
+# the trust hints we receive from kubelet:
+# - If kubelet tags a container workload as untrusted, crio will try first to
+# run it through the untrusted container workload runtime. If it is not set,
+# crio will use the trusted runtime.
+# - If kubelet does not provide any information about the container workload trust
+# level, the selected runtime will depend on the default_container_trust setting.
+# If it is set to "untrusted", then all containers except for the host privileged
+# ones, will be run by the runtime_untrusted_workload runtime. Host privileged
+# containers are by definition trusted and will always use the trusted container
+# runtime. If default_container_trust is set to "trusted", crio will use the trusted
+# container runtime for all containers.
+default_workload_trust = "untrusted"
+
+```
+
+Restart CRI-O to take changes into account
+```
+$ sudo systemctl restart crio
+```
+
+### CRI-containerd
+
+If you picked CRI-containerd, please follow the instructions
+[here](https://github.com/containerd/cri#getting-started-for-developers)
+to get it properly installed.
+
+And for the customization part so that CRI-containerd will pick Kata Containers
```

Comment:
s/And for the customization/To customize CRI-containerd so it picks.../
I think @klynnrif will have a number of style/grammer rewrite feedbacks to maintain consistency with the other docs across the repos.